README.md

```diff
@@ -103,6 +103,32 @@ terminus(server, options);
 server.listen(PORT || 3000);
 ```
 
+## How to set Terminus up with Kubernetes?
+
+When a pod is deleted, Kubernetes will notify it and will wait for `gracePeriod` seconds before killing it.
```

Comment:
nit: `... Kubernetes will notify it and wait for ...` (removes double use of will)